cli/internal/commands/run/form/validation.go

```diff
@@ -152,3 +152,26 @@ func (v *ContainerImage) ValidateTextField(value string) tea.Msg {
 
 	return ValidationMsg("")
 }
+
+var nameRegexp = regexp.MustCompile(`^[a-z\d](?:[-a-z\d]{0,62}[a-z\d])?$`)
+
+type Name struct {
+	Required string
+	Valid    string
+}
+
+func (v *Name) ValidateTextField(value string) tea.Msg {
+	if value == "" {
+		return ValidationMsg(v.Required)
+	}
+
+	if !nameRegexp.MatchString(value) {
+		return ValidationMsg(v.Valid)
+	}
+
```

Comment:
Should we add in a length check (63 char) here as well?

Reply:
It's in the regexp (and I fixed the length, it was validating to a max of 64 characters).